consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/MigrationJob.java

```diff
@@ -371,9 +372,18 @@ protected boolean documentsConsistent(JsonNode sourceDocument, JsonNode destinat
         Iterator<Entry<String, JsonNode>> nodeIterator = sourceDocument.fields();
 
         while (nodeIterator.hasNext()) {
-            Entry<String, JsonNode> sourceNode = nodeIterator.next();
-            if(!getJobConfiguration().getComparisonExclusionPaths().contains(sourceNode.getKey())) {
-                if(!sourceNode.getValue().equals(destinationDocument.findValue(sourceNode.getKey()))) {
+            Entry<String, JsonNode> sourceEntry = nodeIterator.next();
+
+            List<String> excludes = getJobConfiguration().getComparisonExclusionPaths();
+            if(excludes == null || !excludes.contains(sourceEntry.getKey())) {
+                JsonNode sourceNode = sourceEntry.getValue();
+                JsonNode destinationNode = destinationDocument.findValue(sourceEntry.getKey());
+
+                if((JsonNodeType.NULL.equals(sourceNode.getNodeType())
+                        && destinationNode == null)){
+                    continue;
+                }
+                if(!sourceNode.equals(destinationNode)) {
```

Comment:
Seems safer to check that `sourceNode` isn't null, but not sure that it could happen.  Anyway, can't hurt to check for null